apps/backend/src/modules/reviews/review.repository.ts

```diff
@@ -15,23 +20,42 @@ class ReviewRepository implements Repository {
 
 		const review = await this.reviewModel
 			.query()
-			.insert({
-				content,
-				likesCount,
-				poiId,
-				routeId,
-				userId,
-			})
+			.insert({ content, likesCount, poiId, routeId, userId })
 			.returning("*")
 			.execute();
 
 		return ReviewEntity.initialize(review);
 	}
 
-	public async findAll(): Promise<ReviewEntity[]> {
-		const reviews = await this.reviewModel.query().execute();
+	public async findAll(
+		options: null | ReviewSearchQuery,
+	): Promise<ReviewGetByIdResponseDto[]> {
+		const q = this.reviewModel
+			.query()
+			.withGraphJoined("user(selectBasic)")
+			.modifiers({
+				selectBasic(b) {
+					b.select("id", "first_name as firstName", "last_name as lastName");
+				},
```

Comment:
It should work like this 🤔 
```suggestion
				selectUserInfo(builder) {
					builder.select("id", "firstName", "lastName");
				},
```